File: app/src/main/java/com/slidely/photoframe/features/media/fragments/ImageFragment.java
package com.slidely.photoframe.features.media.fragments;

import android.net.Uri;
import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.ImageView;

import com.bumptech.glide.Glide;
import com.slidely.photoframe.R;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;

import static com.slidely.photoframe.features.media.MainActivity.URI_KEY;

public class ImageFragment extends Fragment {
    @Nullable
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container,
                             @Nullable Bundle savedInstanceState) {
        return inflater.inflate(R.layout.image, container, false);
    }

    @Override
    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
        Uri uri = Uri.parse(getArguments().getString(URI_KEY));
        ImageView imageView = (ImageView) getView();
        Glide.with(this)
                .load(uri)
                .into(imageView);
    }
}

File: app/src/main/java/com/k4dima/photoframe/features/main/ui/fragments/ImageFragment.kt
package com.k4dima.photoframe.features.main.ui.fragments

import android.net.Uri
import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.ImageView
import androidx.fragment.app.Fragment
import com.bumptech.glide.Glide
import com.k4dima.photoframe.R
import com.k4dima.photoframe.features.main.ui.MainActivity.Companion.URI_KEY

class ImageFragment : Fragment() {
    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?) =
            inflater.inflate(R.layout.image, container, false)!!

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        val uri = Uri.parse(arguments!!.getString(URI_KEY))
        val imageView = getView() as ImageView
        Glide.with(view)
                .load(uri)
                .into(imageView)
    }
}